build/Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM fedora:32
+FROM centos:8
 
-RUN dnf install -y dnf-plugins-core && \
+RUN dnf install -b -y dnf-plugins-core && \
+    dnf copr enable -y networkmanager/NetworkManager-1.22 && \
     dnf copr enable -y nmstate/nmstate-0.2 && \
-    dnf install -y nmstate iproute iputils && \
+    dnf install -b -y nmstate iproute iputils && \
```

Comment:
-b, --best            try the best available package versions in
                        transactions.

b fails the command if it fails to install ?

Reply:
Try the best available package versions in transactions. Specifically during dnf upgrade, which by default skips over updates that can not be installed for dependency reasons, the switch forces DNF to only consider the latest packages. **When running into packages with broken dependencies, DNF will fail giving a reason why the latest version can not be installed.**

Reply:
thanks, wasnt at the --help but i see its on
https://dnf.readthedocs.io/en/latest/command_ref.html